book/_toc.yml

```diff
@@ -13,3 +13,4 @@ parts:
   - file: markdown
   - file: notebooks
   - file: markdown-notebooks
+  - file: tut2_spec_pd_gpd
```

Comment:
Maybe it's better to add the file names of the JNs or files together within one PR to avoid merge conflicts.

Reply:
Let's just leave it as is, it'll be a good example to learn how to handle merge conflicts :slightly_smiling_face:

Reply:
Ok, first tutorial is merged, so you'll need to resolve the merge conflict here. If you could also rename the file to `tut02_spec_pd_gpd` (add an extra `0`), that would make it consistent with the first tutorial.